qiskit/qobj/pulse_qobj.py

```diff
@@ -322,6 +347,33 @@ def to_dict(self):
             out_dict['header'] = self.header.to_dict()
         return out_dict
 
+    def __repr__(self):
+        instructions_str = [repr(x) for x in self.instructions]
+        instructions_repr = '[' + ', '.join(instructions_str) + ']'
+        out = "PulseQobjExperiment("
+        if hasattr(self, 'config'):
+            out += str(repr(self.config)) + ", "
+        if hasattr(self, 'header'):
+            out += str(repr(self.header)) + ", "
```

Comment:
I think these 2 should have the attribute names included in the string, like `out += "config=" + str(repr(self.config)) + ", "`